draft-packages/table/KaizenDraft/Table/Table.module.scss

```diff
@@ -101,7 +101,13 @@ $row-height-data-variant: 48px;
 
 .headerRowCellTooltipIcon {
   color: $color-blue-500;
-  margin-right: $spacing-xs;
+
+  [dir="ltr"] & {
+    margin-right: $spacing-xs;
```

Comment:
Btw, as we're not supporting IE11 anymore, you should be able to use `margin-inline/block` here instead 😃

Reply:
Thanks @christian314159, I've updated it.